lib/src/sv_onvif.c

```diff
@@ -116,3 +116,23 @@ onvif_media_signing_authenticity_report_free(
     onvif_media_signing_authenticity_t ATTR_UNUSED *authenticity_report)
 {
 }
+
+// Common for Signing and Validation
+
+onvif_media_signing_t* onvif_media_signing_create(MediaSigningCodec ATTR_UNUSED codec)
+{
+    return NULL;
+}
+
+MediaSigningReturnCode onvif_media_signing_reset(onvif_media_signing_t* self)
+{
+  if (self) {
+    return OMS_INVALID_PARAMETER;
+  }
+  return OMS_OK;
```

Comment:
This is a little bit of an odd implementation compared to the others. Why can't it simply be
```
{
  return OMS_NOT_SUPPORTED;
}
```
like all the others? Something looks fishy here.